js/src/components/contentAnalysis/ReadabilityAnalysis.js

```diff
@@ -1,30 +1,26 @@
-/* global wpseoPostScraperL10n wpseoTermScraperL10n */
+/* global wpseoPostScraperL10n wpseoTermScraperL10n wpseoAdminL10n */
 
 import React from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
-import styled from "styled-components";
 import { __ } from "@wordpress/i18n";
+import { utils } from "yoast-components";
 
 import Results from "./Results";
 import Collapsible from "../SidebarCollapsible";
 import getIndicatorForScore from "../../analysis/getIndicatorForScore";
 import { getIconForScore } from "./mapResults";
 
-const AnalysisHeader = styled.span`
-	font-size: 1em;
-	font-weight: bold;
-	margin: 0 0 8px;
-	display: block;
-`;
-
 let localizedData = {};
 if( window.wpseoPostScraperL10n ) {
 	localizedData = wpseoPostScraperL10n;
 } else if ( window.wpseoTermScraperL10n ) {
 	localizedData = wpseoTermScraperL10n;
 }
 
+const { makeOutboundLink } = utils;
+const LearnMoreLink = makeOutboundLink();
```

Comment:
This can be refactored to `const { makeOutboundLink as LearnMoreLink } = utils;`

Reply:
Decided to keep it like this, as I think it is easier to read.